packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -377,25 +383,27 @@ class InspectorTreeController extends Object
     root.children.forEach(_collapseAllNodes);
   }
 
-  int get numRows => root != null ? root.subtreeSize : 0;
+  int get numRows => root?.subtreeSize ?? 0;
 
   int getRowIndex(double y) => max(0, (y - verticalPadding) ~/ rowHeight);
 
-  InspectorTreeRow getRowForNode(InspectorTreeNode node) {
-    return getCachedRow(root.getRowIndex(node));
+  InspectorTreeRow? getRowForNode(InspectorTreeNode node) {
+    return getCachedRow(root!.getRowIndex(node));
   }
 
-  InspectorTreeRow getRow(Offset offset) {
-    if (root == null) return null;
+  InspectorTreeRow? getRow(Offset offset) {
+    final rootLocal = root;
+    if (rootLocal == null) return null;
     final int row = getRowIndex(offset.dy);
-    return row < root.subtreeSize ? getCachedRow(row) : null;
+    return row < rootLocal.subtreeSize ? getCachedRow(row) : null;
   }
 
   void onExpandRow(InspectorTreeRow row) {
     setState(() {
+      final configLocal = config!;
```

Comment:
adds configLocal but dosn't benefit from it.
I assume what you meant to do was add
onExpandLocal.

Reply:
done